Mark 8.0.0-dev

brodybits opened the pull request:
According to the following steps taken, using local checkout of cordova-coho with `improve-patch-and-dev-version-support` branch as proposed in <https://github.com/apache/cordova-coho/pull/188>:

* Update cordova.js from cordova-js@4.2.3, which includes GH-534: Remove obsolete check for JellyBean to work properly on Android Pie:

```sh
./cordova-coho/coho copy-js -r android --js 4.2.3
```

* Update cordova.js from cordova-js@4.2.4, which includes CB-9366 cordova.js log error.stack:

```sh
./cordova-coho/coho copy-js -r android --js 4.2.4
```

* Update cordova.js, version, and VERSION to 8.0.0-dev:

```sh
./cordova-coho/coho prepare-platform-release-branch --version 8.0.0-dev -r android --js 4.2.4
```

Will be merged by push to master.

janpio commented:
Why is a PR with failing tests being merged?
Why does this PR contain a lot more than it says in the PR title?
Why does this PR refer to a WIP and unmerged coho PR?

brodybits commented:
> Why is a PR with failing tests being merged?

It is a build error, no actually failing tests.

I deleted my branch right after merge, that is what caused the failure. I didn't see the failure.

I tried restoring my branch and restarting the build a couple times. Unfortunately it didn't seem to work. It should go green in the next merge.

> Why does this PR contain a lot more than it says in the PR title?

I just updated the title.

Purpose was only to mark 8.0.0-dev. But the procedure involves regenerating cordova.js, and I wanted to do that in a couple smaller steps, to avoid excessive changes combined in a single commit.

> Why does this PR refer to a WIP and unmerged coho PR?

I have some enhancements that help me update cordova.js from a specific cordova-js version. There is quite a bit of review feedback on the PR and I did not a chance to process it. I just updated the description with some more info.

Alternative would be to close the WIP PR on cordova-coho and use my own fork, without giving record of what commands I used to do the update.

I will admit that this update did not go quite right, not much we can do now.
